src/main/kotlin/com/terraformation/backend/customer/api/AdminController.kt

```diff
@@ -210,6 +214,54 @@ class AdminController(
     return "/admin/plantingSite"
   }
 
+  @GetMapping(
+      "/plantingSite/{plantingSiteId}/downloadZone/{plantingZoneId}",
+      produces = [MediaType.APPLICATION_JSON_VALUE])
+  @ResponseBody
+  fun downloadPlantingZone(
+      @PathVariable plantingSiteId: PlantingSiteId,
+      @PathVariable plantingZoneId: PlantingZoneId,
+      response: HttpServletResponse
+  ): Map<String, Any> {
+    response.addHeader(
+        HttpHeaders.CONTENT_DISPOSITION, """attachment; filename="zone-${plantingZoneId}.json"""")
+
+    val zone =
+        plantingSiteStore.fetchSiteById(plantingSiteId).plantingZones.first {
+          it.id == plantingZoneId
+        }
+
+    // Return a GeoJSON document that can be viewed using tools such as geojson.io.
```

Comment:
If we decide to support additional GeoJSON downloads, it'll make sense to refactor this into something more general, but it didn't seem worth doing to me for this one-off use case.